### PR TITLE
few changes, adding Davide Nitti Comments

### PR DESCRIPTION
**Pull request by etienne87**

### CODE REVIEW

File: src/psee_evaluator.py [hunk 1]
```diff
@@ -30,8 +35,9 @@ def main():
     parser = argparse.ArgumentParser(prog='psee_evaluator.py')
     parser.add_argument('gt_folder', type=str, help='GT folder containing .npy files')
     parser.add_argument('dt_folder', type=str, help='RESULT folder containing .npy files')
+    parser.add_argument('--camera', type=str, default='MOOREA', help='ATIS (QVGA) or MOOREA (720p)')
```
Comment (asironi) [inline src/psee_evaluator.py:38]: not sure we can use names as MOOREA and ATIS. (I am quite sure ATIS is wrong, it refers to pixels with EM+CD, independently of resolution) 
to be consitent with paper we should say Gen1 and Gen4

File: src/psee_evaluator.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
         
 
-def evaluate_folders(dt_folder, gt_folder):
+def evaluate_folders(dt_folder, gt_folder, camera):
     dt_file_paths = sorted(glob.glob(dt_folder+'/*'))
```
Comment (dEcmir) [inline src/psee_evaluator.py:13]: maybe filter by extension ?

File: src/psee_evaluator.py [hunk 3]
```diff
@@ -20,6 +20,11 @@ def evaluate_folders(dt_folder, gt_folder):
     result_boxes_list = [reformat_boxes(p) for p in result_boxes_list]
     gt_boxes_list = [reformat_boxes(p) for p in gt_boxes_list]
 
+    min_box_diag = 60 if camera == 'GEN4' else 30
+    min_box_side = 20 if camera == 'GEN1' else 10
+
+    filter_fn = lambda x:filter_boxes(x, int(1e5), min_box_diag, min_box_side)
```
Comment (dEcmir) [inline src/psee_evaluator.py:26]: this doesn't seem to be used,

maybe don't use the map ?

File: src/io/box_filtering.py [hunk 4]
```diff
@@ -6,23 +6,27 @@
...
 
-def filter_boxes(boxes, skip_ts=int(1e5), min_box_diag=30, min_box_side=10):
+def filter_boxes(boxes, skip_ts=int(5e5), min_box_diag=30, min_box_side=10):
     """Filters boxes according to the paper rule. 
 
+    To note: the default represents our threshold when evaluating at VGA resolution (640*360)
+    To note: we assume the initial time of the video is always 0
+
     Args:
         boxes (np.ndarray): structured box array with fields "ts" or "t"
```
Comment (dEcmir) [inline src/io/box_filtering.py:24]: maybe add the other ones ?

File: src/psee_evaluator.py (hunk 3 above)
Comment (etienne87) [reply src/psee_evaluator.py:26]: ah thanks for that catch. the map works okay, if i specify the right function!

File: src/psee_evaluator.py (hunk 1 above)
Comment (etienne87) [reply src/psee_evaluator.py]: done

File: src/io/box_filtering.py [hunk 5]
```diff
@@ -6,23 +6,31 @@
...
 """
 from __future__ import print_function
 import numpy as np
 
 
-def filter_boxes(boxes, skip_ts=int(1e5), min_box_diag=30, min_box_side=10):
+def filter_boxes(boxes, skip_ts=int(5e5), min_box_diag=30, min_box_side=10):
     """Filters boxes according to the paper rule. 
 
+    To note: the default represents our threshold when evaluating at VGA resolution (640*360)
```
Comment (asironi) [inline src/io/box_filtering.py:20]: the default value (30) is for VGA or QGVA (aka Gen1)?

Comment (etienne87) [reply src/io/box_filtering.py:20]: well, it is a bit unclear, perhaps i should remove this default? i know that for Gen4 we use 60, 20, and imerit i think we use 30,10 (but we also used 30,10 for half resolution of Gen4...)

**Pull request merged**